### PR TITLE
Added new venue info.

### PR DESCRIPTION
**Pull request by nikomakela**
PT-832 PT-648 Added 3 new icons (cloned from admin-UI / provider's UI) and added new venue info data to event's occurrence details.

Screenshots and details: https://helsinkisolutionoffice.atlassian.net/browse/PT-832
Palvelutarjotin-admin PR: https://github.com/City-of-Helsinki/palvelutarjotin-admin/pull/153

### CODE REVIEW

File: src/icons/IconToilet.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      version="1.1"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <title>wc_icon</title>
+      <g
+        id="Currents"
+        stroke="none"
+        stroke-width="1"
+        fill="none"
+        fill-rule="evenodd"
```
Comment (Sartonon) [inline src/icons/IconToilet.tsx:20]: Warning: Invalid DOM property `fill-rule`. Did you mean `fillRule`?

**Review by Sartonon — COMMENTED** [review]
Looking good and seems to be working. Should we add some test?

File: src/icons/IconToilet.tsx (hunk 1 above)
Comment (nikomakela) [reply src/icons/IconToilet.tsx]: Fixed

**Comment by nikomakela** [comment]
> Should we add some test?

Yes, something could be added. I'll work with those today.

**Comment by nikomakela** [comment]
I added really simple tests.

File: src/icons/IconPlayIndoor.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      height="24px"
+      viewBox="0 0 24 24"
+      version="1.1"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <title>play_inside_icon</title>
+      <g
+        id="Currents"
+        stroke="none"
+        stroke-width="1"
```
Comment (Sartonon) [inline src/icons/IconPlayIndoor.tsx:18]: This is still giving error: "Warning: Invalid DOM property `stroke-width`. Did you mean `strokeWidth`?"

File: src/icons/IconPlayIndoor.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+
+type Props = { className?: string };
+
+export default function IconPlayIndoor({ className = '' }: Props) {
```
Comment (Sartonon) [inline src/icons/IconPlayIndoor.tsx:5]: Return type should be added to get rid of ts-warning

File: src/icons/IconPlayOutdoor.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+
+type Props = { className?: string };
+
+export default function IconPlayOutdoor({ className = '' }: Props) {
```
Comment (Sartonon) [inline src/icons/IconPlayOutdoor.tsx:5]: Return type should be added to get rid of ts-warning

**Review by Sartonon — APPROVED** [review]
Good

**Pull request merged**